src/drm/fairplay.js

```diff
@@ -33,8 +33,7 @@ export default class FairPlay extends BaseDrmProtocol {
    * @return {boolean} - Whether FairPlay can be play on the current environment.
    */
   static canPlayDrm(drmData: Array<Object>): boolean {
-    FairPlay._logger.debug('Can play DRM scheme of: ' + BaseDrmProtocol.DrmScheme.FAIRPLAY);
```

Comment:
?

Reply:
it's not relevant cause i don't check if I can play I check if source exist and play, so i removed this log from all of them